hammer/technology/sky130/__init__.py

```diff
@@ -46,13 +48,24 @@ def setup_cdl(self) -> None:
         os.makedirs(cache_tech_dir_path, exist_ok=True)
         dest_path = cache_tech_dir_path / f'{self.library_name}.cdl'
 
+        # device names expected in LVS decks
+        if (self.get_setting('vlsi.core.lvs_tool') == "hammer.lvs.calibre"):
+            pmos = 'phighvt'
+            nmos = 'nshort'
```

Comment:
Does Calibre nominally require using the NDA PDK, hence the need to rename the transistors? What about the other Vts?

Reply:
For some reason the only PMOS devices in the CDL netlist are called "pfet_01v8_hvt" and the NMOS are "nfet_01v8", there's no other variations of these devices. But the Calibre LVS deck calls these devices phighvt/nshort.